website/language.py

```diff
@@ -2,7 +2,7 @@
 # -*- coding: utf-8 -*-
 """Various text used throughout the website, e.g. status messages, errors, etc.
 """
-
+from django.utils.translation import ugettext_lazy as _
```

Comment:
I think this is a *great* idea - but I'm uncomfortable with a django method being imported into this file. Internationalization isn't something that we've yet addressed site-wide on the OSF, either. Maybe we're considering it from the start with the API, but if so I'm not aware of that decision having been made.

@brianjgeiger, could you weigh in on this? 

Here's the Django docs on this method: https://docs.djangoproject.com/en/1.8/topics/i18n/translation/#standard-translation